local/check-dependencies.sh

```diff
@@ -34,9 +39,16 @@ function checkDependency() {
   Missing=$(missingCommands $@)
 
   echo -ne "${Name}: "
-  if [ $Missing ]; then failed ; else passed ; fi
+  if [ $Missing ] && [ $AutoInstall ] && [ $AutoInstall == "install" ];
```

Comment:
You don't need the middle clause because all variables will be the empty string until defined. So, the second clause can be `[ "$AutoInstall" == "--install" ]; then`